fantiadl/fantiadl.py

```diff
@@ -79,38 +83,26 @@ def main():
 
     try:
         downloader = FantiaDownloader(session_arg=session_arg, dump_metadata=cmdl_opts.dump_metadata, parse_for_external_links=cmdl_opts.parse_for_external_links, download_thumb=cmdl_opts.download_thumb, directory=cmdl_opts.output_path, quiet=cmdl_opts.quiet, continue_on_error=cmdl_opts.continue_on_error, use_server_filenames=cmdl_opts.use_server_filenames, mark_incomplete_posts=cmdl_opts.mark_incomplete_posts, month_limit=cmdl_opts.month_limit, exclude_file=cmdl_opts.exclude_file, db_path=cmdl_opts.db_path, db_bypass_post_check=cmdl_opts.db_bypass_post_check)
-        if cmdl_opts.download_fanclubs:
-            try:
+
+        # Download based on flags
```

Comment:
Will test this later, I'm a little skittish about raising exceptions but I doubt there's a problem here.